R/bootstrap.R

```diff
@@ -792,9 +792,9 @@ verbatimTextOutput <- function(outputId, placeholder = FALSE) {
 #' @name plotOutput
 #' @rdname plotOutput
 #' @export
-imageOutput <- function(outputId, width = "100%", height="400px",
+imageOutput <- function(outputId, width = "100%", height = "400px",
```

Comment:
Shouldn't this be `auto`?

Reply:
Probably, but I'm not going to mess with it since it would break `renderImage()` returning `height="100%"`